ray-kubernetes/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM ubuntu:latest
+
+# If you wish to default to bash shell instead of sh(Bourne) shell:
+#     RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+
+ENV LANG=C.UTF-8 LC_ALL=C.UTF-8
+
+# disable interactive functions, suppress tzdata prompting for geographic location
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt -qq update \
+    && apt-get -qq install --no-install-recommends apt-utils \
+    && apt-get -qq install git wget \
+    && apt-get -qq install cmake pkg-config build-essential autoconf curl libtool unzip flex bison psmisc nano openssh-server \
+    && apt-get -qq install python-opencv
```

Comment:
We may not need any of these anymore since we aren't compiling ray (other than `curl` which you use below)

Reply:
what kind of minimal functionalities are you expecting? so I know what to get rid of.
It will be cumbersome to install new dependencies seperately on every pod after deployment. and a lot of the rllib test fails without some of the stuff above

Reply:
Sure, things that are intended for convenience like `git` or `wget` are fine. Some of the things are presumably less useful if you aren't compiling Ray, e.g., `flex`/`bison`.

I suppose it's fine to leave those in case someone wants to modify the Dockerfile and actually compile Ray.